training/model_management/environments/model-management/context/requirements.txt

```diff
@@ -2,7 +2,7 @@ azure-core=={{latest-pypi-version}}
 azure-ai-ml=={{latest-pypi-version}}
 azure-identity=={{latest-pypi-version}}
 azureml-evaluate-mlflow=={{latest-pypi-version}}
-ffmpeg=={{latest-pypi-version}}
+ffmpeg-python=={{latest-pypi-version}}
```

Comment:
Is this not a dependency of azureml-evaluate-mlflow that supports whisper?

Reply:
ffmpeg-python is dependency for whisper predict script

Reply:
Ideally, this needs to become a dependency of evaluate-mlflow.

Reply:
Created a bug for this